config/validateRelease.py

```diff
@@ -80,9 +81,18 @@
 with tarfile.open( args.archive, "r:gz" ) as a:
 
 	# getmember still reads the whole archive, so might as well grab them all
-	# as we go. We need to strip the first directory from all paths too as that
-	# contains the release name
-	archivePaths = { os.path.join( *m.name.split( os.sep )[1:] ) for m in a.getmembers() if os.sep in m.name }
+	# as we go. We need to strip the first directory from all paths as that
+	# contains the release name.
+
+	archivePaths = sets.Set()
```

Comment:
Just `set()` should do it.

Reply:
https://github.com/GafferHQ/gaffer/commit/b04ce8615ca9b51406534da83b0b3fc32ac3acd7